### ISSUE TITLE
Run server as a non-privileged user

### ISSUE THREAD

**Issue by thereisnocat** [RaspSDR/server#52]
websdr.bin currently runs as root. Mongoose, which provides the web server that gives access to the SDR, is embedded in websdr.bin. Running a web server as root is suboptimal. It should be possible to update the file in init.d that starts websdr.bin with a `command_user` property that runs everything as a different, non-privileged user. That might require updating some configurations here to place everything somewhere other than the root user’s home directory. This seems like a worthwhile change; when I was a web developer, we went to great lengths to harden web servers, and would never run a web server as root.

**Comment by howard0su** [comment]
the challenge is that normal webserver will not try to configure its hosting environment. But web-888 inherits from kiwi had lots of such features. I am thinking to refector the code to split admin connection with normal user connection. So everything exposed to public should be secure.

However the effort is not small. and it may not be our priority right now.

**Comment by thereisnocat** [comment]
> the challenge is that normal webserver will not try to configure its hosting environment. But web-888 inherits from kiwi had lots of such features. I am thinking to refector the code to split admin connection with normal user connection. So everything exposed to public should be secure.
> 
> However the effort is not small. and it may not be our priority right now.

Fair enough. I appreciate that you take the issue seriously and understand why it would be an issue. I look forward to your eventual update on this.
